### DISCUSSION TITLE
Welcome to Logitech-Z407-Remote-Control-App Discussions!

### DISCUSSION THREAD

**Discussion by Georgi724** [Georgi724/Logitech-Z407-Remote-Control-App#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GaboHuerta** [comment]
Hi Georgi.

Thank you for the application, I'd like to program something simmilar for Android but I imagine you are pretty advanced already in the understanding of the remote, are you willing to share that knowledge?
